src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -173,7 +173,8 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     required_args=[
                         fields.MetricArg(
                             "column",
-                            allowed_columns=constants.SPAN_METRIC_DURATION_COLUMNS,
+                            allowed_columns=constants.SPAN_METRIC_DURATION_COLUMNS
+                            | constants.SPAN_METRIC_COUNT_COLUMNS,
```

Comment:
I don't think this is going to work because you're now allowing gauge metrics and this function is set up to query the distributions table. There are two options here: either update the builder to infer which table to query based on the first letter in the string mri or write a custom query builder function for each of the metrics you'd want to query.

Reply:
https://github.com/getsentry/sentry/pull/69680/files#diff-a7f0127e5f2774e07e683d4994ff7c890e821d3259ad9ce54b1a16733d03ccf0R188
means it's going to query the distributions table